dex/models/_projects/zeroex/bnb/zeroex_bnb_api_fills_uni_v2_pair_creation.sql

```diff
@@ -0,0 +1,35 @@
+{{  config(
+        schema = 'zeroex_bnb',
+        alias = 'fills_uni_v2_pair_creation',
+        materialized='incremental',
+        unique_key = ['pair', 'makerToken', 'takerToken'],
```

Comment:
not sure all three of these should be unique keys, otherwise only `block_time` is left to be updated. likely just `pair`?

Reply:
or potentially this could be append-only or something rather than merge? if so, we'd have to left join `{{ this }}` to make sure not already processed. merge may still be easier with correct keys

Reply:
My understanding of how this works is that you get the latest events for each pair, the old query implied there could be duplicates so we'd want to replace those values with the latest. This could be wrong however.

Reply:
block time might actually not even be needed here

Reply:
i think `pair` should be the only unique key, if we partition by that field in the `rn` and then the maker/taker tokens can change?